changedetectionio/templates/edit.html

```diff
@@ -34,15 +34,18 @@
                     <div class="pure-control-group">
                         {{ render_field(form.minutes_between_check) }}
                         {% if using_default_minutes %}
-                        <span class="pure-form-message-inline">Currently using the <a
+                        <span id="minutes-message-default" class="pure-form-message-inline">Currently using the <a
                                 href="{{ url_for('settings_page', uuid=uuid) }}">default global settings</a>, change to another value if you want to be specific.</span>
                         {% else %}
-                        <span class="pure-form-message-inline">Set to blank to use the <a
+                        <span id="minutes-message-non-default" class="pure-form-message-inline">Set to blank to use the <a
                                 href="{{ url_for('settings_page', uuid=uuid) }}">default global settings</a>.</span>
                         {% endif %}
                     </div>
                     <div class="pure-control-group">
-                        {{ render_field(form.extract_title_as_title) }}
+                        {{ render_field(form.use_seconds, onchange="use_seconds_change()") }}
+                        <span id="seconds-warning" class="pure-form-message-inline" style="display:{{'' if watch.use_seconds == True else 'none'}};">Warning: Setting the recheck duration too short can overload the queue and lead to
```

Comment:
```
>Warning: Setting the recheck duration too short can overload the queue and lead to
unstable performance and application crashes. Test and adjust the duration to ensure
it allows sufficent time to check the target website before queing the watch again.
```

- Actually this is not true, there will be at most only `n` number of workers defined in the `workers` value of the settings (not editable via the UI, default is 10)
- Two spelling mistakes there :)

I think the message should change, it's not that bad, at worst the queue will just grow out to millions and nothing will get checked, better here is to somehow add a check to add a warning if the queue size is greater than 1000 "Hey the check queue is 28,343! things may take a while, or you have too many checks queueing (increase the check time)" etc

Reply:
The system just pushes UUID's into a queue and lets the workers check them in parallel.. so it can be a million or more in that queue and it wouldnt be a problem